Automattic/wp-calypso
# Stats: Add a link to the yesterday's stats page when visiting todays pages

In this PR, I'm using the first approach proposed in #10346 to provide a link to yesterday's stats.

<img width="731" alt="screen shot 2017-01-02 at 10 10 41" src="https://cloud.githubusercontent.com/assets/272444/21586686/d87c5212-d0d3-11e6-839c-b85b194cd5ea.png">

<img width="733" alt="screen shot 2017-01-02 at 10 11 08" src="https://cloud.githubusercontent.com/assets/272444/21586688/dcded7d0-d0d3-11e6-9ad1-5ab0e8586a41.png">

**Open questions**
 - I used a succinct "yesterday" word for the link, Should I add something else or use an icon maybe?
 - Notice the change in the `path` prop for countries stats: the new value matches the URL. This have a small side effect of changing the exported CSV file name, is this problematic? 

closes #10364

**Testing instructions**

 - Navigate throught the "Today" stats's pages and click on the "yesterday" links
 - The links should work properly and redirect to the corresponding stats pages
 - On other stats pages, the "yesterday" link should not be displayed.
